### ISSUE TITLE
Indicate, hide or de-prioritize deprecated projects on the showcase website

### ISSUE THREAD

**Issue by chalu** [acekyd/made-in-nigeria#301]
Projects marked as deprecated on the [repo's README](https://github.com/acekyd/made-in-nigeria) are not so reflected on the [showcase website](https://www.madeinnigeria.dev/projects).

This leads to inconsistency on what is being communicated. and may make it somewhat difficult to decide between two similar projects if you can quickly tell that one is deprecated

**Comment by rotimi-best** [comment]
I agree, I just implemented a PR the checks for archived and deprecated repos then shows it as a chip on the UI.

Ideally we should have like a filter on the projects list to hide archived or deprecated repos.

<img width="1251" alt="image" src="https://github.com/acekyd/made-in-nigeria/assets/31730715/79340f9e-a3cd-4c9e-805b-44fa87d4349a">

@kaf-lamed-beyt what do you think? I am working on a PR with this update.

**Comment by kaf-lamed-beyt** [comment]
LGTM! 

@acekyd, @D-Lite is the color choice okay?

**Comment by acekyd** [comment]
The color choice looks good. On second thought however, a good thing to note is that the criteria for setting projects as deprecated were that the last commit was over 2 years ago (If I remember correctly). This process was also done manually - hopefully there could be an easy way to implement it in future. 

However I'd like to change the intent to something else from deprecated - maybe like not actively maintained or something shorter - cos some of them still work and deprecated might not be the fitting term.

**Comment by D-Lite** [comment]
What do you think of **unmaintained** or **inactive**? @acekyd @kaf-lamed-beyt @chalu

**Comment by kaf-lamed-beyt** [comment]
"Inactive" has a nice ring to it.👌🏽

**Comment by rotimi-best** [comment]
+1 for "Inactive"

In my PR should I rename Deprecated in the README to Inactive?

**Comment by chalu** [comment]
Inactive sounds good to me as well

**Comment by rotimi-best** [comment]
@acekyd @kaf-lamed-beyt @chalu.

Here is my checklist for completion

1. [x] Rename `Deprecated` to `Inactive`. https://github.com/acekyd/made-in-nigeria/pull/310
2. [ ] Implement the parsing of the README on main and then rendering of the tag on each project.

**Comment by kaf-lamed-beyt** [comment]
Hi @rotimi-best, I think, it'll be best if we can sort of automate this, just like @acekyd suggested, instead of manually typing it. 

One way to go about this would be to use the [GitHub Commit REST API](https://docs.github.com/en/rest/commits/commits?apiVersion=2022-11-28) to get the last commit date, and compare if it is in the range @acekyd mentioned &mdash; 2 years in this case. 

The payload's response includes the date, we can use that. 

If you're not going to be able to work on it for now, that's absolutely cool... perhaps @acekyd can merge this PR, then we'd look for a way to work on it internally.

**Comment by rotimi-best** [comment]
@kaf-lamed-beyt It shouldn't be complicated, I can def work on it.

If I am implementing the automated approach of checking if it is archived or inactive, should I just remove the text in the README that mentions whether it is "Inactive" or "Archived" so it doesn't confuse anyone in the future as we add more projects?

**Comment by kaf-lamed-beyt** [comment]
Yes. you should remove it. Since It'll be automated now.

If you have any questions, let me know, I'd be happy to help.

**Comment by acekyd** [comment]
You can proceed to use the local file now @rotimi-best - for the automation check, lets keep the processing time in mind so we don't lock up the main thread and make it all slower running API requests for 200+ projects.

**Comment by acekyd** [comment]
Took only a few years but this has been addressed with github workflows in #346 and #358
